src/me/remie/vulcan/leaguetasks/LeagueScript.java

```diff
@@ -109,6 +109,19 @@ private void checkTasks() {
             }, 1000, 5);
             return;
         }
+
+        // Make sure we have correct filter active.
+        final SimpleWidget dropdownMenuWidget = ctx.widgets.getWidget(657,31).getChild(4);
```

Comment:
Le'ts add a null check to the widget before getting it's 4th child and also null check that child as well.